src/components/PostEditor/PostEditor.js

```diff
@@ -545,12 +546,17 @@ export default class PostEditor extends React.Component {
       'project'
     ].includes(type)
     const canHaveTimes = type !== 'discussion'
-    // Center location autocomplete either on post's current location, or current group's location, or current user's location
+    const params = windowLocation && windowLocation.search ? Object.fromEntries(new URLSearchParams(windowLocation.search)) : null
+    let lat, lng
+    if (params) {
+      lat = params.lat
+      lng = params.lng
+    }
```

Comment:
I think it'd be better if `PostEditor` didn't have to know about the location for this but instead received either `location` or `locationObject` as a prop (or `post.location`) coming from `CreateModal` which handled grabbing the querystring/search params then passed them.  Similarly I think you can handle `currentLocation` by constructing the map lat/lng point into a `locationObject` you pass in as that prop.